pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[build-system]
+# Minimum requirements for the build system to execute, according to PEP518
+# specification.
+requires = ["numpy", "cython", "setuptools"]
```

Comment:
```suggestion
requires = ["numpy", "cython", "setuptools", "wheel"]
```
You also need to add wheel to avoid a warning:
```
  WARNING: Missing build requirements in pyproject.toml for file:///home/tom/work/github/pycortex.
  WARNING: The project does not specify a build backend, and pip cannot fall back to setuptools without 'wheel'.
```

Reply:
Fixed in b5bb9e9673e5b4becff079de0c0e15d321de202a